src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/group_convolution.cpp

```diff
@@ -1349,6 +1433,37 @@ INSTANTIATE_TEST_SUITE_P(smoke_GroupConv_3D_DW_FP32,
                                             ::testing::ValuesIn(fusingParamsSet),
                                             ::testing::Values(empty_plugin_config)),
                          GroupConvolutionLayerCPUTest::getTestCaseName);
+
+const std::vector<std::vector<size_t>> dilations3d_Brdgmm = {{1, 1, 1}};
+const auto groupConvParams_ExplicitPadding_DW_3D_Brdgmm = ::testing::Combine(::testing::ValuesIn(kernels3d),
+                                                                      ::testing::ValuesIn(strides3d),
+                                                                      ::testing::ValuesIn(padBegins3d),
+                                                                      ::testing::ValuesIn(padEnds3d),
+                                                                      ::testing::ValuesIn(dilations3d_Brdgmm),
+                                                                      ::testing::ValuesIn(numOutChannels_DW),
+                                                                      ::testing::ValuesIn(numGroups_DW),
+                                                                      ::testing::Values(ov::op::PadType::EXPLICIT));
+const auto Brdgmm3DCPUSpec = []()-> std::vector<CPUSpecificParams> {
+    if (ov::with_cpu_x86_avx512f()) {
+        return {conv_avx512_dw_3D_nspc_brgconv};
```

Comment:
So what is the point of having `conv_avx2_dw_1D_nspc_brgconv`, `conv_avx2_dw_2D_nspc_brgconv` , etc? 
As I mentioned lets just return CPUSpecificParams{{}, {}, {}, type_str), where type_str is initialized based on ov::with_cpu_x86_avx512f() condition